src/main/java/com/textmessenger/controller/CommentController.java

```diff
@@ -29,27 +29,27 @@
   }
 
   @PostMapping("/post/{id}/user/{userId}")
-  ResponseEntity<?> createComment(@Valid @PathVariable("id") PostRxDto post,
-                                  @Valid @PathVariable("userId") UserRxDto user,
-                                  @Valid @RequestBody CommentRxDto comment) {
+  ResponseEntity<?> createComment(@Valid @PathVariable("id") Post post,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_aSAJjEPbGfqJYu&open=AWWU5_aSAJjEPbGfqJYu)

[ISSUE_KEY]: AWWU5_aSAJjEPbGfqJYu